crates/cxx-qt-gen/test_outputs/inheritance.rs

```diff
@@ -5,6 +5,7 @@ mod inheritance {
         type QModelIndex = cxx_qt_lib::QModelIndex;
         include!("cxx-qt-lib/qvariant.h");
         type QVariant = cxx_qt_lib::QVariant;
+        type QAbstractItemModel;
```

Comment:
why are we not seeing the upcast trait being impl'd in the output? Seems maybe that is going wrong or needs `update_expected` running again?